gradle/libs.versions.toml

```diff
@@ -1,5 +1,5 @@
 [versions]
-android-gradle-plugin = "8.8.0"
+android-gradle-plugin = "8.8.1"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kseito/RewardedTodo/blob/7ea588a971d2f9e4d53a9b15e5a6afd9522dd588/gradle/libs.versions.toml#L2"></span>A newer version of com.android.application than 8.8.1 is available: 8.8.2</td>
    </tr>
  </tbody>
</table>